src/index.js

```diff
@@ -1 +1,68 @@
-import './style.css';
\ No newline at end of file
+import urlApi from './Constance.js';
+import './style.css';
+
+const URL = urlApi();
+const form = document.querySelector('form');
+const name = document.querySelector('input[type="text"]');
+const score = document.querySelector('input[type="number"]');
+const p = document.querySelector('.square');
+const btn = document.querySelector('.btn');
+const gameData = {
+  id: '',
+};
+
+const update = async () => {
```

Comment:
- [ ] Data is not being sent to the API correctly. When a page of the app is refreshed already entered **data vanishes**. Kindly ensure to improve upon this aspect.